spec/runtime/setup_spec.rb

```diff
@@ -108,6 +108,38 @@
     end
   end
 
+  context "load order" do
+    after do
+      ENV.delete "RUBYLIB"
+      ENV.delete "RUBYOPT"
+    end
+
+    it "puts loaded gems after -I and RUBYLIB" do
```

Comment:
please split the `-I` and `RUBYLIB` tests, and add a `:ruby => '> 1.8.7'` filter to the the `-I` test